### ISSUE TITLE
Chpt. 10: Epochs and batch size - batch size won't work

### ISSUE THREAD

**Issue by Dseal95** [ageron/handson-ml2#277]
I have probably made a really small mistake but I am just trying to build the 2 hidden layer MLP classifier on the MNIST fashion set in Chapter 10. 

My training and validation sets are 55,000 and 5,000 sizes respectively. 

When I run the fit() method on my NN I get the following logs:

Epoch 1/30
1719/1719 [==============================] - 1s 812us/step - loss: 0.7237 - accuracy: 0.7644 - val_loss: 0.5207 - val_accuracy: 0.8234

Which is saying that my batch size is 1719 (~3% of training set) - why is the batch size not 55,000? Am I missing something really basic here?

Here are my datasets:

![Screenshot 2020-10-02 at 08 57 20](https://user-images.githubusercontent.com/63999160/94900726-5fdab280-048d-11eb-8a65-4d424a0632c3.png)

**Comment by Dseal95** [comment]
The log of the fit() method for the model in keras has changed. It now shows the number of steps as opposed to the size of the training and validation sets:

New log: 

Epoch 1/30
1719/1719 [==============================] - 1s 812us/step - loss: 0.7237 - accuracy: 0.7644 - val_loss: 0.5207 - val_accuracy: 0.8234 

{'verbose': 1, 'epochs': 30, 'steps': 1719}

Old Log:

Train on 55000 samples, validate on 5000 samples
Epoch 1/30
55000/55000 [==============================] - 2s 44us/sample - loss: 0.7226 - accuracy: 0.7641 - val_loss: 0.5073 - val_accuracy: 0.8320
